core/src/com/fallenflame/game/GameEngine.java

```diff
@@ -373,7 +373,9 @@ public void update(float delta) {
         }
         if (sneakPressed && !sneakPrevious) {
             // If player just started sneaking
-            level.makeSneak();
+            sneakPressed = level.makeSneak();
+            // Sets sneakPressed to makeSneak() return fo sneakPressed is false if player could not sneak
```

Comment:
I believe we said the player can still sneak even if they are "out", but instead the ghost would just come

Reply:
Oh whoops, that's left over code. I will remove it (but the behavior now is as you describe)